Compute/tests/System/V1/SmokeTest.php

```diff
@@ -49,25 +53,38 @@ public static function setUpBeforeClass(): void
             self::fail('Environment variable PROJECT_ID must be set for smoke test');
         }
         self::$instancesClient = new InstancesClient();
+        self::$zoneOperationsClient = new ZoneOperationsClient();
         self::$machineType = sprintf(
             'https://www.googleapis.com/compute/v1/projects/%s/zones/%s/machineTypes/n1-standard-1',
             self::$projectId,
             self::ZONE
         );
     }
 
+    public function setUp(): void
+    {
+        self::$name = "gapicphp" . strval(rand($min = 100000, $max = 999999));
+        self::$dirty = false;
+    }
+
+    public function tearDown(): void
+    {
+        if (self::$dirty == true) {
+            self::$instancesClient->delete(self::$name, self::$projectId, self::ZONE);
+        }
+    }
+
     public static function tearDownAfterClass(): void
     {
         self::$instancesClient->close();
     }
 
-    public function testInsertInstance()
+    private function insertInstance(): void
     {
-        $name = "gapicphp" . strval(rand($min = 100000, $max = 999999));
         $disk = new AttachedDisk([
             'boot' => true,
-             "auto_delete" => true,
-             "type" => 0,
+            "auto_delete" => true,
+            "type" => 1,
```

Comment:
this is an enum, and `1` is not a valid type. See [AttachedDisk\Type](https://github.com/googleapis/google-cloud-php/blob/master/Compute/src/V1/AttachedDisk/Type.php#L21). This should be something like this instead:

```php
"type" => AttachedDisk\Type::PERSISTENT,
```